packages/yoroi-extension/app/UI/features/notifications/module/NotificationCenter.tsx

```
@@ -24,6 +25,46 @@ const StyledButton = styled(IconButton)(({ theme }: any) => ({
   },
 }));
 
+const Notification = (props: { notification: (typeof appState.notifications.all)[number] }) => {
+  const { notification } = props;
+  const navigateTo = useNavigate();
+  const { setIsNotificationCenterOpen } = useNotifications();
+  const onNavigate = () => {
+    if (!notification.redirection) {
```

Should we throw an exception here? Is there any notification without redirection? Maybe have only an if here?

---

If this function runs, then this notification must have redirection.